InstrumentationEngine.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio Version 16
-VisualStudioVersion = 16.0.28727.205
+# Visual Studio Version 17
+VisualStudioVersion = 17.2.32201.480
```

Comment:
I know I mentioned this in the other PR, but what's the minimum VS version we can build this on? Ideally we can build it on the public stable dev 17 version.

Reply:
We should be able to use VS2019 since our current toolset is v142 and winsdk is 10. I'll get this reverted

Reply:
FWIW, it doesn't seem like this header does anything. I was able to open and build using the public d17.1 version using the supported toolset version.